chain/chain/src/chain.rs

```diff
@@ -729,15 +727,15 @@ impl Chain {
                 {
                     return Err(ErrorKind::InvalidChunk.into());
                 }
-            } else if chunk_header.height_created() == block.header().height() {
-                if !runtime_adapter.verify_chunk_header_signature(
+            } else if chunk_header.height_created() == block.header().height()
+                && !runtime_adapter.verify_chunk_header_signature(
```

Comment:
TBH, I am not a huge fan of blindly merging `if cond_a { if cond_b }`. Often times, the second form is more readable.

Reply:
OTOH, it can also be the case that the second form hides a bug, and in this sense it might be good to normalize all ifs as a rule of thumb.